app/package.json

```diff
@@ -77,7 +77,7 @@
     "postcss-preset-env": "^7.7.2",
     "prettier": "^2.6.2",
     "prettier-plugin-sort-json": "^0.0.3",
-    "react-number-format": "^4.9.4",
+    "react-number-format": "^5.0.1",
```

Comment:
Notes for reviewers: Upgraded to [v5](https://s-yadav.github.io/react-number-format/docs/migration) in order to get access to some of the nicer features like being able to access the unformatted string value.